server/jetstream_cluster.go

```diff
@@ -949,6 +951,9 @@ func (js *jetStream) monitorCluster() {
 		beenLeader   bool
 	)
 
+	// Highwayhash key for generating hashes.
+	key := sha256.Sum256([]byte("clusters"))
```

Comment:
Can just create random bytes here, or if you prefer just use sha256 vs HH? HH is quick.